cmd/minikube/cmd/start.go

```diff
@@ -85,6 +85,11 @@ const (
 	kvmQemuURI            = "kvm-qemu-uri"
 	kvmGPU                = "kvm-gpu"
 	kvmHidden             = "kvm-hidden"
+	minikubeEnvPrefix     = "MINIKUBE"
+	defaultEmbedCerts     = false
```

Comment:
Unless they are used elsewhere, inline all default flag values directly to where the flag is defined.